app/javascript/src/components/Dashboard/Notes/NoteCard.jsx

```diff
@@ -1,54 +1,59 @@
 import React from "react";
 
 import { Clock, MenuVertical } from "@bigbinary/neeto-icons";
-// @ts-ignore
 import { Typography, Avatar, Tag, Dropdown } from "neetoui";
 
 const NoteCard = ({
   title,
   body,
   createdAt,
   tags = [{ tag: "Getting Started" }],
-}) => (
-  <div className="neeto-ui-border-gray-300 my-4 flex flex-col border-2 px-1">
-    <div className="p-2">
-      <div className="mt-2 flex justify-between px-2">
-        <Typography className="" style="h3">
-          {title}
+  setSelectedNoteIds,
+  setShowDeleteAlert,
+  id,
+}) => {
+  const handleDeleteNote = () => {
+    setShowDeleteAlert(true);
+    setSelectedNoteIds([id]);
+  };
+  return (
```

Comment:
You can add a newline before the `return` statement. It helps in visually segregating between method declarations and exit statements. Here is segregates out the presentation logic from others.